rxandroid/src/main/java/rx/android/content/ContentObservable.java

```diff
@@ -13,21 +13,45 @@
  */
 package rx.android.content;
 
+import android.content.ContentResolver;
 import android.content.Context;
 import android.content.Intent;
 import android.content.IntentFilter;
 import android.content.SharedPreferences;
+import android.net.Uri;
 import android.database.Cursor;
 import android.os.Handler;
 
 import rx.Observable;
 
+import static android.os.Build.VERSION.SDK_INT;
+
 public final class ContentObservable {
     private ContentObservable() {
         throw new AssertionError("No instances");
     }
 
     /**
+     * Create Observable that wraps ContentObserver and emits received uri.
+     * Need API level 16 or above. If API level below 16, UnsupportedOperationException
+     * will be threw.
+     *
+     * @param contentResolver
+     * @param uri The URI to watch for changes. This can be a specific row URI, or a base URI
+     * for a whole class of content.
+     * @param schedulerHandler Handler identifying the thread that will observe
+     *      the content changes.
+     */
+    static Observable<Uri> fromContentObserver(final ContentResolver contentResolver,
+        final Uri uri, final Handler schedulerHandler) {
+        if (SDK_INT < 16) {
```

Comment:
Meh, you're right. That's annoying. I guess this could be reversed to `SDK_INT > VERSION_CODES.ICE_CREAM_SANDWICH_MR1`, not sure if that feels weird though.